### PR TITLE
Trapezoidal distribution

### PR DESCRIPTION
**Pull request by ethankward**
Add trapezoidal distribution to crv_types.

### CODE REVIEW

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -2276,6 +2278,84 @@ def StudentT(name, nu):
...
+                  \frac{2(d-x)}{(d-c)(d+c-a-b)} & \mathrm{for\ } c \le x < d, \\
+                  0 & \mathrm{for\ } d < x.
+                \end{cases}
+
+    Parameters
+    ==========
+
+    a : Real number, :math:`a \in \left(-\infty, \infty\right)`
+    b : Real number, :math:`b \geq a`
+    c : Real number, :math:`c \geq d`
```
Comment (gxyd) [inline sympy/stats/crv_types.py]: `\leq`?

File: sympy/stats/crv_types.py [hunk 2]
```diff
@@ -2276,6 +2278,84 @@ def StudentT(name, nu):
...
+                  \frac{2(x-a)}{(b-a)(d+c-a-b)} & \mathrm{for\ } a \le x < b, \\
+                  \frac{2}{d+c-a-b} & \mathrm{for\ } b \le x < c, \\
+                  \frac{2(d-x)}{(d-c)(d+c-a-b)} & \mathrm{for\ } c \le x < d, \\
+                  0 & \mathrm{for\ } d < x.
+                \end{cases}
+
+    Parameters
+    ==========
+
+    a : Real number, :math:`a \in \left(-\infty, \infty\right)`
```
Comment (gxyd) [inline sympy/stats/crv_types.py]: I would prefer writing them as written on wikipedia article i.e.

  | a                  (         a         <         d         )                 {\displaystyle a\;(a<d)}     - lower bound                         b                  (         a         ≤         b         <         c         )                 {\displaystyle b\;(a\leq b<c)}     - level start                         c                  (         b         <         c         ≤         d         )                 {\displaystyle c\;(b<c\leq d)}     - level end                         d                  (         c         ≤         d         )                 {\displaystyle d\;(c\leq d)}     - upper bound
-- | --

**Comment by gxyd** [comment]
Can you add test cases for these as well?

**Comment by ethankward** [comment]
Sure, I'll add some tests. Should I add a check method for the arguments? I noticed that only a few of the distributions had it (uniform and triangular do not) so I wasn't sure if it was necessary.

**Comment by gxyd** [comment]
I don't think it would be necessary for "Trapezoidal Distribution". In general, when the parameters have satisfies comparison with "non-symbol", like in "Beta Distribution" `alpha > 0` and `beta > 0` both (alpha and beta) have comparison with `0`, i.e why they are included in `check`, while with "Trapezoidal Distribution" it is comparison `b >= a` (which for symbols wouldn't be evaluated), so we don't add `check` method. So it wouldn't required for "Trapezoidal Distribution".

There are a few distributions in `crv_types` for which `check` is missing (ex. BeniniDistribution), but that can be fixed in separate PR if you want to.

File: sympy/stats/crv_types.py (hunk 2 above)
Comment (gxyd) [reply sympy/stats/crv_types.py]: Only this comment needs to be addressed. Apart from that this PR look good.

**Comment by gxyd** [comment]
Looks good to me. Will merge when it passes the tests.

**Comment by gxyd** [comment]
Thanks @ethankward merging.

**Pull request merged**